Makefile

```diff
@@ -414,7 +414,13 @@ create-workload-cluster: $(ENVSUBST)
 	timeout --foreground 600 bash -c "while ! kubectl --kubeconfig=./kubeconfig get nodes | grep master; do sleep 1; done"
 
 	# Deploy calico
-	kubectl --kubeconfig=./kubeconfig apply -f templates/addons/calico.yaml
+	@if [[ "${CLUSTER_TEMPLATE}" == *ipv6* ]]; then \
```

Comment:
this is temporary until cluster resource set is enabled

Reply:
I might not understand resource set.  Is this still required?  I thought manual apply of cni's is no longer required?

Reply:
It is. I enabled CRS for e2e tests only in this PR. The rest will happen in #947 which I'm holding to include the ipv6 stuff once this one merges.